### ISSUE TITLE
zonedTimeToUtc not working for numbers

### ISSUE THREAD

**Issue by SimonPedley** [marnusw/date-fns-tz#115]
Hi all!

I am having trouble converting to UTC from a timezoned date in the form of a number.
Below is some output from Google Chrome's console.

```
> date_fns_1.formatRFC3339(date_fns_tz_1.zonedTimeToUtc(time, props.timeZone))
< "2021-03-13T15:00:00Z"
> date_fns_1.formatRFC3339(date_fns_tz_1.zonedTimeToUtc(new Date(time), props.timeZone))
< "2021-03-13T14:00:00Z"
> time
< 1615647600000
```
`time` is a number that has previously been converted to the `Europe/Paris` (+1:00) timezone.
When i convert the number to the `Date` object before passing it to `zonedTimeToUtc()` it works and i get the correct UTC time from it. But passing just the number, i get back the timezoned value.

I've taken a quick peek at the source code and it appears that if the date parameter is a number, it just converts it to a date and passes it back without doing any timezone convertion.

I've also checked this for the `utcToZonedTime()` and numbers are correctly treated just the same as date objects.
I've checked that i am on the latest version:
```
"date-fns": "^2.19.0",
"date-fns-tz": "^1.1.3"
```

Am i misunderstanding something here or have i stumbled on a bug?

Thanks for your help!

**Comment by ankit-gaur** [comment]
whenever you pass a number to the method its treat simply as time elapsed from 1970UTC i.e. if you pass number then timezone won't matter and utc time will always be same irrespective of timezone you specify.

**Comment by marnusw** [comment]
This should be fixed in `1.2.2`. Please reopen if not.
